src/main/java/seedu/mtracker/console/AddEtfParser.java

```diff
@@ -2,11 +2,42 @@
 
 import seedu.mtracker.commands.AddEtfCommand;
 import seedu.mtracker.commands.AddInstrumentCommand;
+import seedu.mtracker.commands.AddStockCommand;
+import seedu.mtracker.ui.TextUi;
 
 public class AddEtfParser extends AddInstrumentParser {
 
+    public static String INSTRUMENT_TYPE = "etf";
+
+    public String getEtfRemarksFromUser() {
+        TextUi.displayAddRemarksInstruction();
+        return getUserInput();
+    }
+
+    public String getEtfpastReturnsFromUser() {
+        TextUi.displayAddPastReturnsInstruction();
+        return getUserInput();
+    }

```

Comment:
There should be validation here to check that input is a number, pls create a separate issue for this or resolve in this pr directly